refinery/tool_manager/tests.py

```diff
@@ -255,10 +255,8 @@ def setUp(self):
         self.tool_defs_url_root = '/api/v2/tool_definitions/'
 
         self.mock_parameter = ParameterFactory(
-            name="Test Param",
-            description="Test Param Description",
-            value_type=Parameter.STRING,
-            default_value="Coffee"
+            name="Test Param", description="Test Param Description",
```

Comment:
The left format is easier to read. Why the change?  https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#source-formatting

Reply:
It's customary in Python to list function arguments with no extra new lines (example: https://www.python.org/dev/peps/pep-0008/#indentation). Also, unnecessary newlines make functions/methods longer and more difficult to read (i.e., to fit on one screen).